model/security/get_action_group/structures.smithy

```diff
@@ -0,0 +1,20 @@
+//  SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure GetActionGroup_Input{
+    @required
+    @httpLabel
+    action_group: String
+}
+
+@output
+structure GetActionGroup_Output {
+    action_group: Action_Group
```

Comment:
This operation returns a map:
```suggestion
    @httpPayload
    content: ActionGroupMap
```

It is expected that this will cause build errors until #127 is merged